integration-tests/bats/performance.bats

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+BATS_TEST_TIMEOUT=50
+
+# This function was used to create the dolt repo used for this test. It is not run during testing.
+create_repo() {
+    dolt init
+    dolt checkout -b full
+
+    dolt sql -q 'create table t (pk int primary key, c0 text default "1", c3 text default "2", c4 text default "3", c5 text default "4", c6 text default "5", c7 text default "6");'
+    dolt commit -Am "new table t"
+
+    echo "insert into t(pk) values" > import.sql
+    for i in {1..100000}
+    do
+        echo "  ($i)," >> import.sql
+    done
+    echo "  (104857);" >> import.sql
+
+    dolt sql < import.sql
```

Comment:
(minor) I don't think you need to check in the generated import.sql script. I know I had suggested checking in a sql file to create the table, but since you've got the create_repo function in the BATS, I think that's an even better way to get reproducibility, so I think you can safely remove the checked-in import.sql file since we can easily regenerate the test database from the BATS function you made. You could also trim down some other files, for example branch_control.db shouldn't be needed since we aren't using branch permissions.

Reply:
I didn't mean to check in the .sql script. Removed.